cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  {"id": "cpp.veles", "name": "veles", "tech": ["dl"], "tags": ["framework"], "repo": "Samsung/veles",
+  "imports": ["veles.h"], "examples": ["https://github.com/Samsung/veles/tree/master/libVeles/tests"]},
+  {"id": "cpp.kaldi", "name": "Kaldi", "tech": ["speech-recognition"], "tags": ["toolkit"], "repo": "kaldi-asr/kaldi",
+  "imports": ["base/kaldi-common.h"], "examples": ["https://github.com/kaldi-asr/kaldi/tree/master/egs"]},
+  
+  {"id": "cpp.asio", "name": "Asio", "tech":["networking"], "tags":["async-io", "async-networking"], "repo": "chriskohlhoff/asio",
+  "imports": ["asio.hpp"], "examples": ["https://github.com/chriskohlhoff/asio/tree/master/asio/src/examples"]},
+  {"id": "cpp.boost-asio", "name": "Boost.Asio", "tech":["networking"], "tags":["async-io", "async-networking"], "repo": "chriskohlhoff/asio",
+  "imports": ["asio.hpp"], "examples": ["https://github.com/chriskohlhoff/asio/tree/master/asio/src/examples", "http://think-async.com/"]},
+  {"id": "cpp.actor-framework", "name": "Actor Framework", "tags": ["concurrency", "framework"], "tech": ["faster-code"], "repo": "actor-framework/actor-framework",
+  "imports": ["caf/all.hpp"], "examples": ["https://github.com/actor-framework/actor-framework/tree/master/examples"]},
+  {"id": "cpp.libev", "name": "libev", "tags":["event", "async"], "tech": ["networking"], "repo": "http://software.schmorp.de/pkg/libev.html",
+  "imports": ["ev.h", "ev++.h"], "examples": ["http://pod.tst.eu/http://cvs.schmorp.de/libev/ev.pod"]},
+  {"id": "cpp.libevent", "name": "libevent", "tags":["event", "async"], "tech": ["networking"], "repo": "libevent/libevent",
+  "imports": ["event2/event.h", "event2/dns.h", "event2/util.h", "event2/listener.h", "event2/buffer.h", "event2/bufferevent.h"],
+  "examples": ["https://github.com/libevent/libevent/tree/master/sample"]},
+  {"id": "cpp.libuv", "name": "libuv", "tech":["networking"], "tags":["async-io"], "repo": "libuv/libuv",
+  "imports": ["uv.h"], "examples": ["https://github.com/libuv/libuv/tree/v1.x/samples"]},
+  {"id": "cpp.uvw", "name": "uvw", "tech":["networking"], "tags":["async-io"], "repo": "skypjack/uvw",
+  "imports": ["uvw.hpp"], "examples": ["https://github.com/skypjack/uvw#code-example"]},
+
+  {"id": "cpp.kfr", "name": "kfr", "tech": ["computational-science"], "tags": ["dsp", "audio", "fft"], "repo": "kfrlib/kfr",
+  "imports": ["kfr/base.hpp", "kfr/dft.hpp", "kfr/dsp.hpp", "kfr/io.hpp"], "examples": ["https://github.com/kfrlib/kfr/tree/master/examples"]},
+  {"id": "cpp.maximilian", "name": "Maximilian", "tech": ["computational-science"], "tags": ["dsp", "audio", "fft"], "repo": "micknoise/Maximilian",
+  "imports": ["maximilian.h", "libs/maxim.h"], "examples": ["https://github.com/micknoise/Maximilian/tree/master/maximilian_examples"]},
+  {"id": "cpp.openal-soft", "name": "OpenAL Soft", "tech": ["audio"], "tags": ["3D audio", "audio-synthesis", "audio-game"], "repo": "kcat/openal-soft",
```

Comment:
There may be a little bit of a confusion with tags. Usually, there should be just one tag, but if you really feel like you need multiple tags, please make sure it describes the library is comprehensively as possible. The reason is we don't have real estate to show multiple tags per library on a profile; we intend to only show first. 

When I look at your multi-tag entries, they seem mostly fine, but please take another look, and keep in mind that the first tag ideally should be good enough to describe the whole package.

Reply:
Got it